Bottom: bc2d4f2ec8f6697f9f35af406826f461df658b16
Top:    beccceb74fb428213c4a359d156b9e0070d2a6f3
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-04 13:55:29 +0800

Refresh of virtio-introduce-packed-ring

---

diff --git a/include/uapi/linux/virtio_config.h b/include/uapi/linux/virtio_config.h
index 947f6a3..274425b 100644
--- a/include/uapi/linux/virtio_config.h
+++ b/include/uapi/linux/virtio_config.h
@@ -75,6 +75,7 @@
  */
 #define VIRTIO_F_IOMMU_PLATFORM		33
 
+/* Packed virtqueue layout */
 #define VIRTIO_F_RING_PACKED		34
 
 /*
